conf.py

```diff
@@ -17,19 +17,21 @@
 # -- Project information -----------------------------------------------------
 import time
 
-project = 'FT2zXcXv1'
-author = 'autor'
+project = 'Sprawozdanie, Laboratorium - Fizyka II'
+author = 'Autorzy:[AUTORZY]'
 copyright = '2024, ' + author
 titlepage = {
-        "faculty": "Fizyki i Informatyki Stosowanej",
```

Comment:
also, I'll revert this - I'm too lazy to change this every time. Set this in your fork :smile: